setup.m

```diff
@@ -0,0 +1,23 @@
+% Run this file once to set up your Matlab session for Neuron interaction.
+
+% User setting:
+NeuronInstallationDirectory = 'C:\nrn';
+
+% All dependencies of the generated interface library must be findable.
+% WINDOWS: Put them on the PATH
+dllpath = fullfile(NeuronInstallationDirectory, 'bin');

```

Comment:
Already have a check here (on one file), that indeed a correct installation directory, with the expected dll files.

Reply:
Resolved in latest commit.